src/actions/signupSigninActions.js

```diff
@@ -1,39 +1,35 @@
-import * as types from './types';
+import { notify } from 'react-notify-toast';
+import axios from 'axios';
+import baseURL from '../utils/urls';
```

Comment:
Move this file to the `helpers` folder and rename it to `config.js`. Export an object containing baseUrl property in the config file instead